application/app/Http/Requests/GetClassifierValuesRequest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Enums\ClassifierValueType;
+
+class GetClassifierValuesRequest extends ApiFormRequest
+{
+    public function getType(): ?ClassifierValueType
+    {
+        return ClassifierValueType::tryFrom($this->input('type'));
+    }
+
+    public function rules(): array
+    {
+        return [
+            'type' => ['string', 'in:'.implode(',', ClassifierValueType::values())],
```

Comment:
**[non-blocker]** Since the `ClassifierValueService->search()` function allows for `type` to be `null`, shouldn’t the `type` be [`nullable`](https://laravel.com/docs/10.x/validation#a-note-on-optional-fields) here as well? Correct me if I‘m wrong.